schemas/thing.py

```diff
@@ -20,6 +20,41 @@
 from schemas import BaseCreateModel, BaseUpdateModel, BaseResponseModel
 from schemas.location import LocationResponse
 
+# -------- VALIDATE ----------
+
+
+class ValidateWell(BaseModel):
+    well_depth: float | None = None  # in feet
```

Comment:
```suggestion
    well_depth_in_feet: float | None = Field(None, alias="well_depth")
```

This change would improve clarity by using `_in_feet` to make units explicit, reducing ambiguity and preventing any future unit-mixing errors, while the alias preserves database compatibility. It also makes the API more 'self-documenting.'

Reply:
Rather than add the unit to the field name I can add another field called `well_depth_unit` and have it automatically be set to `ft`. Awhile ago we made the decision to not include units in the names, so I'll proceed with this method to keep a consistent style.

Reply:
Fields can have descriptions, so I'll add that to document it for create/update